include/votca/tools/graphdistvisitor.h

```diff
@@ -57,8 +57,8 @@ class GraphDistVisitor : public Graph_BF_Visitor {
   /// Note the only manipulation to the BF visitor is the need to add a
   /// distance attribute to each of the graph nodes.
   void exploreNode(std::pair<Index, GraphNode>& p_gn, Graph& g,
```

Comment:
exploreNode is not a very descriptive name, considering nearly all arguments are not const